src/pick/__init__.py

```diff
@@ -34,6 +34,7 @@ class Picker(Generic[OPTION_T]):
     selected_indexes: List[int] = field(init=False, default_factory=list)
     index: int = field(init=False, default=0)
     screen: Optional["curses._CursesWindow"] = None
+    position: dict = field(default_factory={"y0": 1, "x0": 1})
```

Comment:
A simple dict is hard to use. People will have to check the documentation to find out which key they should use, and the type checker and IDE won't provide much help.

Since there are only two fields to describe a position, and it wouldn't be changed very often, I suggest using a Position = namedtuple('Position', ['x', 'y']) as this type, for users can create it as a normal class, or just passing a tuple as they like.

Reply:
that makes sense, I'll see what I can do when I have the time, feel free to add those changed in yourself as well

Reply:
Apologies if this is obvious, but how/where do I implement your suggestion?  I believe that lines 37, 191, 132, 117, 115 would be affected.

If I understand you correctly, you are saying change the current `dict` to a tuple? I'm not too sure on the exact intricacies of how the arguments in this program work, are you able to provide me with a pointer of what to do?

Reply:
I mean add a new global type `Position = namedtuple('Position', ['x', 'y'])`, and use it in these lines instead of a dict.

Reply:
I'm not sure why use a `x0` name instead of `x` in current branch, is there any reason for this? If so, please point it out and keep the name.

Reply:
Okay thank you, is `namedtuple` meant to be literally that, or should I name it something?

I wrote this a while ago, not sure why I chose `x0`.

Reply:
I mean https://docs.python.org/3/library/collections.html#collections.namedtuple

I think `x` / `y` is a more direct name if we have no special reasons.

Reply:
Ah thank you  :)

Yes, I agree the names should be changed. I will investigate this evening.

Reply:
I have implemented the `namedtuple` changes you suggested, however it throws the following error:

```
TypeError: Picker._start() missing 1 required positional argument: 'position'
```

I believe I have added it as optional (by providing a default value?) so I'm not sure what it going wrong here

Reply:
No, it's not optional parameter: https://github.com/aisk/pick/pull/95/files#diff-65b1705380373a054a015132e8b066afb61ec4172a411962ece35a8cf768d48cR170

And I see some codes are still using `dict` as postion type, these need to be updated too.

Reply:
okay I think I get it now, thanks  :)
I will update the PR in a bit

Reply:
Regarding the linked line, the `screen` argument doesn't appear to be "conventionally" optional (the same as `position`) but is optional?

From my understanding, the [definition of the `pick` function](https://github.com/aisk/pick/blob/1c08d5fb56edd941040183102f8cf0d9c6c8c3bd/src/pick/__init__.py#L194) handles the optional-ness of arguments by providing a default value, as does the [definition of the `Picker` class](https://github.com/aisk/pick/blob/1c08d5fb56edd941040183102f8cf0d9c6c8c3bd/src/pick/__init__.py#L40) similarly.

I will commit the changes from `dict` to `Position` in other function definitions but I still am unsure why it isn't working

Reply:
I think you don't pass the position argument to the `_start` function, which is a callback to `curses.warpper`. You can pass it like `curses.wrapper(self._start, position)`, see the document: https://docs.python.org/3/library/curses.html#curses.wrapper

Reply:
thank you, that is all fixed now if you wish to review :)